docs/modules/ROOT/pages/how-tos/create-fluentd-cert.adoc

```diff
@@ -0,0 +1,53 @@
+= Create Fluentd Certificate
+
+To create an SSL certificate for the s3-forwarder use this command and answer the questions like in the example below.
```

Comment:
```suggestion
To create a TLS certificate for the s3-forwarder use this command and answer the questions like in the example below.
```
I think the term SSL shouldn't be used anymore, except if it it's part of a name like OpenSSL.